gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -123,22 +130,23 @@ def print_help(self):
             if self.source != ""
             else "\nSource: ?\n"
         )
+        help_text += self.price_str
         help_text += """
+
     chart       view a candle chart for a specific cryptocurrency
     headlines   crypto sentiment from 15+ major news headlines [Finbrain]
```

Comment:
also just noticed these lines don't have same indentation as above, would be good if they matched

Reply:
![image](https://user-images.githubusercontent.com/43375532/148195682-0b937a0c-a97c-4eed-9882-24ac2e4f781b.png)

i tried to follow what was done in stocks context, should i move chart headlines etc to the beginning of the line?

Reply:
No, I didn't mean putting them at the beginning. I meant that if you look at the indentation of chart/headlines it doesn't match the number of spaces of the ones below of the menus

Reply:
oh my bad, should be good now